### PR TITLE
Feature: Sass lint

### PR DESCRIPTION
**Pull request by tiagomapmarques**

### CODE REVIEW

File: src/api/config.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import { CLIEngine as Eslint } from 'eslint';
+import { Configuration as StylelintConfiguration } from 'stylelint';
+
+import * as tsConfig from '../../tsconfig.json';
+import { compile } from '../compile';
+
+const CONFIG_DEFAULT = `${__dirname}/../../index.js`;
+
+export interface Config {
+  logic: Eslint.LintReport;
```
Comment (tiagomapmarques) [inline src/api/config.ts]: Most likely the wrong typing

Comment (tiagomapmarques) [reply src/api/config.ts]: Fixed

File: src/api/lint.ts [hunk 2]
```diff
@@ -1,84 +1,38 @@
...
-    // eslint-disable-next-line no-console
-    console.log('SCSS Linting: Not implemented yet…');
-
-    result.style = {};
+  if (parsedOptions.typescript || parsedOptions.javascript) {
+    linterPromises.push(lintLogic(config, parsedOptions));
   }
 
-  return result;
+  return new Promise((resolve): Promise<LintResult | void> => Promise.all(linterPromises)
```
Comment (claudivieira) [inline src/api/lint.ts]: Could you explain me why are you using a new promise to resolve another promise?

Comment (tiagomapmarques) [reply src/api/lint.ts]: Old leftover code. Fixed.

**Pull request merged**